### ISSUE TITLE
fix: Firefox setup warning when accessibility is already enabled

### ISSUE THREAD

**Issue by renjfk** [renjfk/SimplyTrack#67]
### What happened?

Firefox website tracking can show the setup warning even when `accessibility.force_disabled` is already set to `-1` in Firefox `about:config`.

Follow-up to #60.

### Expected behavior

SimplyTrack should not tell users to enable Firefox accessibility setup again when it is already enabled. If Firefox URL lookup fails for another reason, the message should reflect that.

### Steps to reproduce

1. Set `accessibility.force_disabled` to `-1` in Firefox `about:config`.
2. Use Firefox website tracking.
3. See the Firefox setup warning.

### macOS version

Unknown

### App version

0.9-pr65.c09a3b5 (8)

### Browser involved

Firefox

### Screenshots

See the latest comment on #60.

**Comment by renjfk** [comment]
Hi @xverges, could you check if [`v0.9-pr68.8db3058`](https://github.com/renjfk/SimplyTrack/releases/tag/v0.9-pr68.8db3058) works?

**Comment by xverges** [comment]
🎉 
Sites tracked. No warning on the UI. Nothing displayed by `log stream --predicate 'subsystem == "com.renjfk.SimplyTrack"' --style compact`
